processing file asmFiles/aluop_and_alubranch.asm (1 of 36)
processing file asmFiles/beq_notTaken.asm (2 of 36)
processing file asmFiles/beq_taken.asm (3 of 36)
processing file asmFiles/bge_notTaken.asm (4 of 36)
processing file asmFiles/bge_taken.asm (5 of 36)
processing file asmFiles/bgeu_notTaken.asm (6 of 36)
processing file asmFiles/bgeu_taken.asm (7 of 36)
processing file asmFiles/blt_notTaken.asm (8 of 36)
processing file asmFiles/blt_taken.asm (9 of 36)
processing file asmFiles/bltu_notTaken.asm (10 of 36)
processing file asmFiles/bltu.taken.asm (11 of 36)
processing file asmFiles/bne_notTaken.asm (12 of 36)
processing file asmFiles/bne_taken.asm (13 of 36)
processing file asmFiles/branch_hazard2.asm (14 of 36)
processing file asmFiles/branch_hazard.asm (15 of 36)
processing file asmFiles/count_days.asm (16 of 36)
--- memsim.hex	2026-10-01 19:26:48.364149493 -0400
+++ memcpu.hex	2026-10-01 19:26:48.359149544 -0400
@@ -39,6 +39,9 @@
 :04002600FFC1011302
 :040027000081202311
 :0400280000008067ED
-:043FFD000000016D52
-:043FFE000000251288
+:043E73000000252CFA
+:043FDF000000016D70
+:043FE0000000001AC3
+:043FFD000000001EA2
+:043FFE0000000008B7
 :00000001FF
processing file asmFiles/fetch_and_instr.asm (17 of 36)
processing file asmFiles/fib.asm (18 of 36)
--- memsim.hex	2026-10-01 19:26:50.826124442 -0400
+++ memcpu.hex	2026-10-01 19:26:50.821124493 -0400
@@ -15,26 +15,5 @@
 :04000E00FFFFFFFFF2
 :0400210000000001DA
 :0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/halt.asm (19 of 36)
--- memsim.hex	2026-10-01 19:26:52.069111794 -0400
+++ memcpu.hex	2026-10-01 19:26:52.064111845 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :040001000030202388
 :04000200FFFFFFFFFE
 :040003000000019365
processing file asmFiles/jal.asm (20 of 36)
--- memsim.hex	2026-10-01 19:26:53.334098923 -0400
+++ memcpu.hex	2026-10-01 19:26:53.328098984 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :040001000030202388
 :040002000080026F09
 :04000300FFFFFFFFFD
processing file asmFiles/jalr.asm (21 of 36)
--- memsim.hex	2026-10-01 19:26:54.519086865 -0400
+++ memcpu.hex	2026-10-01 19:26:54.514086916 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :040001000030202388
 :0400020001400313A3
 :04000300000302678D
processing file asmFiles/mergesort.asm (22 of 36)
--- memsim.hex	2026-10-01 19:26:56.104070737 -0400
+++ memcpu.hex	2026-10-01 19:26:56.100070778 -0400
@@ -86,28 +86,28 @@
 :0400550000078463B9
 :04005600FE9FF06FAA
 :0400570000008067BE
-:0400C00000000040FC
+:0400C0000000000438
 :0400C100000000013A
 :0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
+:0400C3000000000732
+:0400C4000000000B2D
+:0400C5000000000C2B
+:0400C6000000001521
+:0400C700000000161F
 :0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
+:0400C900000000191A
+:0400CA000000001B17
+:0400CB000000001D14
+:0400CC000000002010
+:0400CD00000000210E
+:0400CE000000002905
+:0400CF0000000033FA
+:0400D00000000036F6
+:0400D10000000038F3
+:0400D2000000003DED
 :0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
+:0400D4000000003EEA
+:0400D50000000040E7
 :0400D60000000042E4
 :0400D70000000047DE
 :0400D8000000004BD9
@@ -119,101 +119,82 @@
 :0400DE000000005AC4
 :0400DF0000000063BA
 :0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
 :0401400000000001BA
 :0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0401420000000007B2
+:040143000000000BAD
+:040144000000000CAB
+:0401450000000015A1
+:04014600000000169F
+:04014700000000169E
+:04014800000000199A
+:040149000000001B97
+:04014A000000001D94
+:04014B000000002090
+:04014C00000000218E
+:04014D000000002985
+:04014E00000000307D
+:040151000000004268
+:040152000000006445
+:040153000000000F99
+:040154000000005C4B
+:040155000000004165
+:040156000000002085
+:04015700000000099B
+:040158000000002F74
+:040159000000005949
+:04015A000000001190
+:04015B000000000799
+:04015C00000000237C
+:04015D00000000445A
+:04015E00000000207D
+:04015F000000000A92
+:040160000000000794
+:040161000000001783
+:040162000000005C3D
+:040163000000005B3D
+:04016400000000286F
+:040165000000001A7C
+:04016600000000088D
+:040167000000002470
+:04016800000000266D
+:04016900000000088A
+:04016A00000000266B
+:04016B000000001080
+:04016C00000000325D
+:04016D000000000787
+:04016E00000000434A
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (23 of 36)
--- memsim.hex	2026-10-01 19:26:57.389057662 -0400
+++ memcpu.hex	2026-10-01 19:26:57.385057703 -0400
@@ -21,6 +21,7 @@
 :04001400FFC1011314
 :040015000081202323
 :0400160000008067FF
+:043FFA000000000FB4
 :043FFD0000000004BC
-:043FFE000000000CB3
+:043FFE0000000003BC
 :00000001FF
processing file asmFiles/mult_procedure.asm (24 of 36)
--- memsim.hex	2026-10-01 19:26:58.712044200 -0400
+++ memcpu.hex	2026-10-01 19:26:58.707044251 -0400
@@ -29,7 +29,8 @@
 :04001C00FFC101130C
 :04001D00008120231B
 :04001E0000008067F7
+:043FF80000000018AD
 :043FFC0000000005BC
-:043FFD0000000014AC
-:043FFE000000003C83
+:043FFD0000000004BC
+:043FFE0000000003BC
 :00000001FF
processing file asmFiles/other.asm (25 of 36)
processing file asmFiles/read_after_write.asm (26 of 36)
processing file asmFiles/search.asm (27 of 36)
--- memsim.hex	2026-10-01 19:27:02.648004153 -0400
+++ memcpu.hex	2026-10-01 19:27:02.643004204 -0400
@@ -14,7 +14,7 @@
 :04000D00FE0004E30A
 :04000E0000D12023DA
 :04000F00FFFFFFFFF1
-:0400200000000124B7
+:0400200000000128B3
 :0400210000005C6F10
 :040022000000006476
 :040023000000087D54
processing file asmFiles/temptest.asm (28 of 36)
processing file asmFiles/test.branchjump1.asm (29 of 36)
--- memsim.hex	2026-10-01 19:27:04.977980459 -0400
+++ memcpu.hex	2026-10-01 19:27:04.972980510 -0400
@@ -16,6 +16,7 @@
 :04000F00004526235F
 :04001000FEB094E3C7
 :04001100FFFFFFFFEF
+:040020000000BA5CC6
+:040021000000BA5CC5
 :0400220000000024B6
-:040024000000003C9C
 :00000001FF
processing file asmFiles/test.ldst1.asm (30 of 36)
processing file asmFiles/test.ldst2.asm (31 of 36)
processing file asmFiles/test.loadstore.asm (32 of 36)
processing file asmFiles/test.loaduse.asm (33 of 36)
processing file asmFiles/test.rtype.asm (34 of 36)
processing file asmFiles/write_after_read.asm (35 of 36)
processing file asmFiles/write_after_write.asm (36 of 36)
processing file asmFiles/aluop_and_alubranch.asm (1 of 36)
processing file asmFiles/beq_notTaken.asm (2 of 36)
processing file asmFiles/beq_taken.asm (3 of 36)
processing file asmFiles/bge_notTaken.asm (4 of 36)
processing file asmFiles/bge_taken.asm (5 of 36)
processing file asmFiles/bgeu_notTaken.asm (6 of 36)
processing file asmFiles/bgeu_taken.asm (7 of 36)
processing file asmFiles/blt_notTaken.asm (8 of 36)
processing file asmFiles/blt_taken.asm (9 of 36)
processing file asmFiles/bltu_notTaken.asm (10 of 36)
processing file asmFiles/bltu.taken.asm (11 of 36)
processing file asmFiles/bne_notTaken.asm (12 of 36)
processing file asmFiles/bne_taken.asm (13 of 36)
processing file asmFiles/branch_hazard2.asm (14 of 36)
processing file asmFiles/branch_hazard.asm (15 of 36)
processing file asmFiles/count_days.asm (16 of 36)
--- memsim.hex	2026-10-01 19:57:34.524474073 -0400
+++ memcpu.hex	2026-10-01 19:57:34.520474113 -0400
@@ -39,6 +39,9 @@
 :04002600FFC1011302
 :040027000081202311
 :0400280000008067ED
-:043FFD000000016D52
-:043FFE000000251288
+:043E73000000252CFA
+:043FDF000000016D70
+:043FE0000000001AC3
+:043FFD000000001EA2
+:043FFE0000000008B7
 :00000001FF
processing file asmFiles/fetch_and_instr.asm (17 of 36)
processing file asmFiles/fib.asm (18 of 36)
--- memsim.hex	2026-10-01 19:57:36.961449439 -0400
+++ memcpu.hex	2026-10-01 19:57:36.956449490 -0400
@@ -15,26 +15,5 @@
 :04000E00FFFFFFFFF2
 :0400210000000001DA
 :0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/halt.asm (19 of 36)
--- memsim.hex	2026-10-01 19:57:38.086438068 -0400
+++ memcpu.hex	2026-10-01 19:57:38.080438128 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :040001000030202388
 :04000200FFFFFFFFFE
 :040003000000019365
processing file asmFiles/jal.asm (20 of 36)
--- memsim.hex	2026-10-01 19:57:39.394424846 -0400
+++ memcpu.hex	2026-10-01 19:57:39.389424897 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :040001000030202388
 :040002000080026F09
 :04000300FFFFFFFFFD
processing file asmFiles/jalr.asm (21 of 36)
--- memsim.hex	2026-10-01 19:57:40.539413272 -0400
+++ memcpu.hex	2026-10-01 19:57:40.534413323 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :040001000030202388
 :0400020001400313A3
 :04000300000302678D
processing file asmFiles/mergesort.asm (22 of 36)
--- memsim.hex	2026-10-01 19:57:42.119397301 -0400
+++ memcpu.hex	2026-10-01 19:57:42.113397362 -0400
@@ -86,28 +86,28 @@
 :0400550000078463B9
 :04005600FE9FF06FAA
 :0400570000008067BE
-:0400C00000000040FC
+:0400C0000000000438
 :0400C100000000013A
 :0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
+:0400C3000000000732
+:0400C4000000000B2D
+:0400C5000000000C2B
+:0400C6000000001521
+:0400C700000000161F
 :0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
+:0400C900000000191A
+:0400CA000000001B17
+:0400CB000000001D14
+:0400CC000000002010
+:0400CD00000000210E
+:0400CE000000002905
+:0400CF0000000033FA
+:0400D00000000036F6
+:0400D10000000038F3
+:0400D2000000003DED
 :0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
+:0400D4000000003EEA
+:0400D50000000040E7
 :0400D60000000042E4
 :0400D70000000047DE
 :0400D8000000004BD9
@@ -119,101 +119,82 @@
 :0400DE000000005AC4
 :0400DF0000000063BA
 :0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
 :0401400000000001BA
 :0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0401420000000007B2
+:040143000000000BAD
+:040144000000000CAB
+:0401450000000015A1
+:04014600000000169F
+:04014700000000169E
+:04014800000000199A
+:040149000000001B97
+:04014A000000001D94
+:04014B000000002090
+:04014C00000000218E
+:04014D000000002985
+:04014E00000000307D
+:040151000000004268
+:040152000000006445
+:040153000000000F99
+:040154000000005C4B
+:040155000000004165
+:040156000000002085
+:04015700000000099B
+:040158000000002F74
+:040159000000005949
+:04015A000000001190
+:04015B000000000799
+:04015C00000000237C
+:04015D00000000445A
+:04015E00000000207D
+:04015F000000000A92
+:040160000000000794
+:040161000000001783
+:040162000000005C3D
+:040163000000005B3D
+:04016400000000286F
+:040165000000001A7C
+:04016600000000088D
+:040167000000002470
+:04016800000000266D
+:04016900000000088A
+:04016A00000000266B
+:04016B000000001080
+:04016C00000000325D
+:04016D000000000787
+:04016E00000000434A
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (23 of 36)
--- memsim.hex	2026-10-01 19:57:43.350384858 -0400
+++ memcpu.hex	2026-10-01 19:57:43.346384898 -0400
@@ -21,6 +21,7 @@
 :04001400FFC1011314
 :040015000081202323
 :0400160000008067FF
+:043FFA000000000FB4
 :043FFD0000000004BC
-:043FFE000000000CB3
+:043FFE0000000003BC
 :00000001FF
processing file asmFiles/mult_procedure.asm (24 of 36)
--- memsim.hex	2026-10-01 19:57:44.692371293 -0400
+++ memcpu.hex	2026-10-01 19:57:44.686371354 -0400
@@ -29,7 +29,8 @@
 :04001C00FFC101130C
 :04001D00008120231B
 :04001E0000008067F7
+:043FF80000000018AD
 :043FFC0000000005BC
-:043FFD0000000014AC
-:043FFE000000003C83
+:043FFD0000000004BC
+:043FFE0000000003BC
 :00000001FF
processing file asmFiles/other.asm (25 of 36)
processing file asmFiles/read_after_write.asm (26 of 36)
processing file asmFiles/search.asm (27 of 36)
--- memsim.hex	2026-10-01 19:57:48.647331315 -0400
+++ memcpu.hex	2026-10-01 19:57:48.642331366 -0400
@@ -14,7 +14,7 @@
 :04000D00FE0004E30A
 :04000E0000D12023DA
 :04000F00FFFFFFFFF1
-:0400200000000124B7
+:0400200000000128B3
 :0400210000005C6F10
 :040022000000006476
 :040023000000087D54
processing file asmFiles/temptest.asm (28 of 36)
processing file asmFiles/test.branchjump1.asm (29 of 36)
--- memsim.hex	2026-10-01 19:57:51.123306287 -0400
+++ memcpu.hex	2026-10-01 19:57:51.118306338 -0400
@@ -16,6 +16,7 @@
 :04000F00004526235F
 :04001000FEB094E3C7
 :04001100FFFFFFFFEF
+:040020000000BA5CC6
+:040021000000BA5CC5
 :0400220000000024B6
-:040024000000003C9C
 :00000001FF
processing file asmFiles/test.ldst1.asm (30 of 36)
processing file asmFiles/test.ldst2.asm (31 of 36)
processing file asmFiles/test.loadstore.asm (32 of 36)
processing file asmFiles/test.loaduse.asm (33 of 36)
processing file asmFiles/test.rtype.asm (34 of 36)
processing file asmFiles/write_after_read.asm (35 of 36)
processing file asmFiles/write_after_write.asm (36 of 36)
processing file asmFiles/jal.asm (1 of 2)
--- memsim.hex	2026-10-01 20:07:23.303514707 -0400
+++ memcpu.hex	2026-10-01 20:07:23.298514757 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :040001000030202388
 :040002000080026F09
 :04000300FFFFFFFFFD
processing file asmFiles/jalr.asm (2 of 2)
--- memsim.hex	2026-10-01 20:07:24.627501293 -0400
+++ memcpu.hex	2026-10-01 20:07:24.622501344 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :040001000030202388
 :0400020001400313A3
 :04000300000302678D
processing file asmFiles/aluop_and_alubranch.asm (1 of 36)
processing file asmFiles/beq_notTaken.asm (2 of 36)
processing file asmFiles/beq_taken.asm (3 of 36)
processing file asmFiles/bge_notTaken.asm (4 of 36)
processing file asmFiles/bge_taken.asm (5 of 36)
processing file asmFiles/bgeu_notTaken.asm (6 of 36)
processing file asmFiles/bgeu_taken.asm (7 of 36)
processing file asmFiles/blt_notTaken.asm (8 of 36)
processing file asmFiles/blt_taken.asm (9 of 36)
processing file asmFiles/bltu_notTaken.asm (10 of 36)
processing file asmFiles/bltu.taken.asm (11 of 36)
processing file asmFiles/bne_notTaken.asm (12 of 36)
processing file asmFiles/bne_taken.asm (13 of 36)
processing file asmFiles/branch_hazard2.asm (14 of 36)
processing file asmFiles/branch_hazard.asm (15 of 36)
processing file asmFiles/count_days.asm (16 of 36)
--- memsim.hex	2026-10-01 20:07:46.748277183 -0400
+++ memcpu.hex	2026-10-01 20:07:46.744277224 -0400
@@ -39,6 +39,9 @@
 :04002600FFC1011302
 :040027000081202311
 :0400280000008067ED
-:043FFD000000016D52
-:043FFE000000251288
+:043E73000000252CFA
+:043FDF000000016D70
+:043FE0000000001AC3
+:043FFD000000001EA2
+:043FFE0000000008B7
 :00000001FF
processing file asmFiles/fetch_and_instr.asm (17 of 36)
processing file asmFiles/fib.asm (18 of 36)
--- memsim.hex	2026-10-01 20:07:49.354250781 -0400
+++ memcpu.hex	2026-10-01 20:07:49.349250832 -0400
@@ -15,26 +15,5 @@
 :04000E00FFFFFFFFF2
 :0400210000000001DA
 :0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/halt.asm (19 of 36)
--- memsim.hex	2026-10-01 20:07:50.579238371 -0400
+++ memcpu.hex	2026-10-01 20:07:50.575238411 -0400
@@ -1,4 +1,4 @@
-:0400000000000001FB
+:0400000000000002FA
 :040001000030202388
 :04000200FFFFFFFFFE
 :040003000020019345
processing file asmFiles/jal.asm (20 of 36)
--- memsim.hex	2026-10-01 20:07:51.779226214 -0400
+++ memcpu.hex	2026-10-01 20:07:51.775226254 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :040001000030202388
 :040002000080026F09
 :04000300FFFFFFFFFD
processing file asmFiles/jalr.asm (21 of 36)
--- memsim.hex	2026-10-01 20:07:53.008213762 -0400
+++ memcpu.hex	2026-10-01 20:07:53.004213803 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :040001000030202388
 :0400020001400313A3
 :04000300000302678D
processing file asmFiles/mergesort.asm (22 of 36)
--- memsim.hex	2026-10-01 20:07:54.600197634 -0400
+++ memcpu.hex	2026-10-01 20:07:54.594197695 -0400
@@ -86,28 +86,28 @@
 :0400550000078463B9
 :04005600FE9FF06FAA
 :0400570000008067BE
-:0400C00000000040FC
+:0400C0000000000438
 :0400C100000000013A
 :0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
+:0400C3000000000732
+:0400C4000000000B2D
+:0400C5000000000C2B
+:0400C6000000001521
+:0400C700000000161F
 :0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
+:0400C900000000191A
+:0400CA000000001B17
+:0400CB000000001D14
+:0400CC000000002010
+:0400CD00000000210E
+:0400CE000000002905
+:0400CF0000000033FA
+:0400D00000000036F6
+:0400D10000000038F3
+:0400D2000000003DED
 :0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
+:0400D4000000003EEA
+:0400D50000000040E7
 :0400D60000000042E4
 :0400D70000000047DE
 :0400D8000000004BD9
@@ -119,101 +119,82 @@
 :0400DE000000005AC4
 :0400DF0000000063BA
 :0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
 :0401400000000001BA
 :0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0401420000000007B2
+:040143000000000BAD
+:040144000000000CAB
+:0401450000000015A1
+:04014600000000169F
+:04014700000000169E
+:04014800000000199A
+:040149000000001B97
+:04014A000000001D94
+:04014B000000002090
+:04014C00000000218E
+:04014D000000002985
+:04014E00000000307D
+:040151000000004268
+:040152000000006445
+:040153000000000F99
+:040154000000005C4B
+:040155000000004165
+:040156000000002085
+:04015700000000099B
+:040158000000002F74
+:040159000000005949
+:04015A000000001190
+:04015B000000000799
+:04015C00000000237C
+:04015D00000000445A
+:04015E00000000207D
+:04015F000000000A92
+:040160000000000794
+:040161000000001783
+:040162000000005C3D
+:040163000000005B3D
+:04016400000000286F
+:040165000000001A7C
+:04016600000000088D
+:040167000000002470
+:04016800000000266D
+:04016900000000088A
+:04016A00000000266B
+:04016B000000001080
+:04016C00000000325D
+:04016D000000000787
+:04016E00000000434A
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (23 of 36)
--- memsim.hex	2026-10-01 20:07:55.934184119 -0400
+++ memcpu.hex	2026-10-01 20:07:55.929184170 -0400
@@ -21,6 +21,7 @@
 :04001400FFC1011314
 :040015000081202323
 :0400160000008067FF
+:043FFA000000000FB4
 :043FFD0000000004BC
-:043FFE000000000CB3
+:043FFE0000000003BC
 :00000001FF
processing file asmFiles/mult_procedure.asm (24 of 36)
--- memsim.hex	2026-10-01 20:07:57.335169925 -0400
+++ memcpu.hex	2026-10-01 20:07:57.330169976 -0400
@@ -29,7 +29,8 @@
 :04001C00FFC101130C
 :04001D00008120231B
 :04001E0000008067F7
+:043FF80000000018AD
 :043FFC0000000005BC
-:043FFD0000000014AC
-:043FFE000000003C83
+:043FFD0000000004BC
+:043FFE0000000003BC
 :00000001FF
processing file asmFiles/other.asm (25 of 36)
processing file asmFiles/read_after_write.asm (26 of 36)
processing file asmFiles/search.asm (27 of 36)
--- memsim.hex	2026-10-01 20:08:01.540127324 -0400
+++ memcpu.hex	2026-10-01 20:08:01.534127385 -0400
@@ -14,7 +14,7 @@
 :04000D00FE0004E30A
 :04000E0000D12023DA
 :04000F00FFFFFFFFF1
-:0400200000000124B7
+:0400200000000128B3
 :0400210000005C6F10
 :040022000000006476
 :040023000000087D54
processing file asmFiles/temptest.asm (28 of 36)
processing file asmFiles/test.branchjump1.asm (29 of 36)
--- memsim.hex	2026-10-01 20:08:04.123101155 -0400
+++ memcpu.hex	2026-10-01 20:08:04.117101216 -0400
@@ -16,6 +16,7 @@
 :04000F00004526235F
 :04001000FEB094E3C7
 :04001100FFFFFFFFEF
+:040020000000BA5CC6
+:040021000000BA5CC5
 :0400220000000024B6
-:040024000000003C9C
 :00000001FF
processing file asmFiles/test.ldst1.asm (30 of 36)
processing file asmFiles/test.ldst2.asm (31 of 36)
processing file asmFiles/test.loadstore.asm (32 of 36)
processing file asmFiles/test.loaduse.asm (33 of 36)
processing file asmFiles/test.rtype.asm (34 of 36)
processing file asmFiles/write_after_read.asm (35 of 36)
processing file asmFiles/write_after_write.asm (36 of 36)
processing file asmFiles/halt.asm (1 of 1)
--- memsim.hex	2026-10-01 20:09:18.935343218 -0400
+++ memcpu.hex	2026-10-01 20:09:18.930343269 -0400
@@ -1,4 +1,4 @@
-:0400000000000001FB
+:0400000000000002FA
 :040001000030202388
 :04000200FFFFFFFFFE
 :040003000020019345
processing file asmFiles/halt.asm (1 of 1)
processing file asmFiles/halt.asm (1 of 1)
processing file asmFiles/aluop_and_alubranch.asm (1 of 36)
processing file asmFiles/beq_notTaken.asm (2 of 36)
processing file asmFiles/beq_taken.asm (3 of 36)
processing file asmFiles/bge_notTaken.asm (4 of 36)
processing file asmFiles/bge_taken.asm (5 of 36)
processing file asmFiles/bgeu_notTaken.asm (6 of 36)
processing file asmFiles/bgeu_taken.asm (7 of 36)
processing file asmFiles/blt_notTaken.asm (8 of 36)
processing file asmFiles/blt_taken.asm (9 of 36)
processing file asmFiles/bltu_notTaken.asm (10 of 36)
processing file asmFiles/bltu.taken.asm (11 of 36)
processing file asmFiles/bne_notTaken.asm (12 of 36)
processing file asmFiles/bne_taken.asm (13 of 36)
processing file asmFiles/branch_hazard2.asm (14 of 36)
processing file asmFiles/branch_hazard.asm (15 of 36)
processing file asmFiles/count_days.asm (16 of 36)
--- memsim.hex	2026-10-01 20:13:33.573761171 -0400
+++ memcpu.hex	2026-10-01 20:13:33.568761221 -0400
@@ -39,6 +39,9 @@
 :04002600FFC1011302
 :040027000081202311
 :0400280000008067ED
-:043FFD000000016D52
-:043FFE000000251288
+:043E73000000252CFA
+:043FDF000000016D70
+:043FE0000000001AC3
+:043FFD000000001EA2
+:043FFE0000000008B7
 :00000001FF
processing file asmFiles/fetch_and_instr.asm (17 of 36)
processing file asmFiles/fib.asm (18 of 36)
--- memsim.hex	2026-10-01 20:13:36.095735592 -0400
+++ memcpu.hex	2026-10-01 20:13:36.089735653 -0400
@@ -15,26 +15,5 @@
 :04000E00FFFFFFFFF2
 :0400210000000001DA
 :0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
 :0403C0000000001623
 :00000001FF
processing file asmFiles/halt.asm (19 of 36)
processing file asmFiles/jal.asm (20 of 36)
--- memsim.hex	2026-10-01 20:13:38.682709354 -0400
+++ memcpu.hex	2026-10-01 20:13:38.676709414 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :040001000030202388
 :040002000080026F09
 :04000300FFFFFFFFFD
processing file asmFiles/jalr.asm (21 of 36)
--- memsim.hex	2026-10-01 20:13:40.004695945 -0400
+++ memcpu.hex	2026-10-01 20:13:39.999695996 -0400
@@ -1,4 +1,3 @@
-:0400000000000001FB
 :040001000030202388
 :0400020001400313A3
 :04000300000302678D
processing file asmFiles/mergesort.asm (22 of 36)
--- memsim.hex	2026-10-01 20:13:41.707678673 -0400
+++ memcpu.hex	2026-10-01 20:13:41.702678724 -0400
@@ -86,28 +86,28 @@
 :0400550000078463B9
 :04005600FE9FF06FAA
 :0400570000008067BE
-:0400C00000000040FC
+:0400C0000000000438
 :0400C100000000013A
 :0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
+:0400C3000000000732
+:0400C4000000000B2D
+:0400C5000000000C2B
+:0400C6000000001521
+:0400C700000000161F
 :0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
+:0400C900000000191A
+:0400CA000000001B17
+:0400CB000000001D14
+:0400CC000000002010
+:0400CD00000000210E
+:0400CE000000002905
+:0400CF0000000033FA
+:0400D00000000036F6
+:0400D10000000038F3
+:0400D2000000003DED
 :0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
+:0400D4000000003EEA
+:0400D50000000040E7
 :0400D60000000042E4
 :0400D70000000047DE
 :0400D8000000004BD9
@@ -119,101 +119,82 @@
 :0400DE000000005AC4
 :0400DF0000000063BA
 :0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
 :0401400000000001BA
 :0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:0401420000000007B2
+:040143000000000BAD
+:040144000000000CAB
+:0401450000000015A1
+:04014600000000169F
+:04014700000000169E
+:04014800000000199A
+:040149000000001B97
+:04014A000000001D94
+:04014B000000002090
+:04014C00000000218E
+:04014D000000002985
+:04014E00000000307D
+:040151000000004268
+:040152000000006445
+:040153000000000F99
+:040154000000005C4B
+:040155000000004165
+:040156000000002085
+:04015700000000099B
+:040158000000002F74
+:040159000000005949
+:04015A000000001190
+:04015B000000000799
+:04015C00000000237C
+:04015D00000000445A
+:04015E00000000207D
+:04015F000000000A92
+:040160000000000794
+:040161000000001783
+:040162000000005C3D
+:040163000000005B3D
+:04016400000000286F
+:040165000000001A7C
+:04016600000000088D
+:040167000000002470
+:04016800000000266D
+:04016900000000088A
+:04016A00000000266B
+:04016B000000001080
+:04016C00000000325D
+:04016D000000000787
+:04016E00000000434A
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (23 of 36)
--- memsim.hex	2026-10-01 20:13:42.891666664 -0400
+++ memcpu.hex	2026-10-01 20:13:42.886666715 -0400
@@ -21,6 +21,7 @@
 :04001400FFC1011314
 :040015000081202323
 :0400160000008067FF
+:043FFA000000000FB4
 :043FFD0000000004BC
-:043FFE000000000CB3
+:043FFE0000000003BC
 :00000001FF
processing file asmFiles/mult_procedure.asm (24 of 36)
--- memsim.hex	2026-10-01 20:13:44.206653327 -0400
+++ memcpu.hex	2026-10-01 20:13:44.201653378 -0400
@@ -29,7 +29,8 @@
 :04001C00FFC101130C
 :04001D00008120231B
 :04001E0000008067F7
+:043FF80000000018AD
 :043FFC0000000005BC
-:043FFD0000000014AC
-:043FFE000000003C83
+:043FFD0000000004BC
+:043FFE0000000003BC
 :00000001FF
processing file asmFiles/other.asm (25 of 36)
processing file asmFiles/read_after_write.asm (26 of 36)
processing file asmFiles/search.asm (27 of 36)
--- memsim.hex	2026-10-01 20:13:48.327611531 -0400
+++ memcpu.hex	2026-10-01 20:13:48.322611581 -0400
@@ -14,7 +14,7 @@
 :04000D00FE0004E30A
 :04000E0000D12023DA
 :04000F00FFFFFFFFF1
-:0400200000000124B7
+:0400200000000128B3
 :0400210000005C6F10
 :040022000000006476
 :040023000000087D54
processing file asmFiles/temptest.asm (28 of 36)
processing file asmFiles/test.branchjump1.asm (29 of 36)
--- memsim.hex	2026-10-01 20:13:51.036584055 -0400
+++ memcpu.hex	2026-10-01 20:13:51.031584106 -0400
@@ -16,6 +16,7 @@
 :04000F00004526235F
 :04001000FEB094E3C7
 :04001100FFFFFFFFEF
+:040020000000BA5CC6
+:040021000000BA5CC5
 :0400220000000024B6
-:040024000000003C9C
 :00000001FF
processing file asmFiles/test.ldst1.asm (30 of 36)
processing file asmFiles/test.ldst2.asm (31 of 36)
processing file asmFiles/test.loadstore.asm (32 of 36)
processing file asmFiles/test.loaduse.asm (33 of 36)
processing file asmFiles/test.rtype.asm (34 of 36)
processing file asmFiles/write_after_read.asm (35 of 36)
processing file asmFiles/write_after_write.asm (36 of 36)
